src/frontend/src/catalog/system_catalog/pg_catalog/pg_type.rs

```diff
@@ -56,6 +56,9 @@ pub const PG_TYPE_DATA: &[(i32, &str)] = &[
     (20, "int8"),
     (21, "int2"),
     (23, "int4"),
+    // Note: rw doesn't support `text` type, returning it is just a workaround to be compatible
+    // with PostgreSQL.
```

Comment:
tbh our `varchar` is more like pg text rather than pg varchar, which accepts an optional max length not supported by pg text or our `varchar` 😂